examples/elasticsearch/README.md

```diff
@@ -42,48 +43,83 @@ Then install the sample Elasticsearch application with the release name `my-rele
 `es-test` using the command below. Make sure you have the kanister controller running in namespace `kasten-io` which is the default setting in Elasticsearch charts. Otherwise, you will also have to set the `kanister.controller_namespace` parameter value to the respective kanister controller namespace in the following command:
 
 ```bash
-$ kubectl create namespace es-test
-$ helm install --namespace es-test elasticsearch elastic/elasticsearch --set antiAffinity=soft
+$ helm install --namespace es-test elasticsearch elastic/elasticsearch --set antiAffinity=soft --create-namespace
 ```
 
-The command deploys Elasticsearch on the Kubernetes cluster in the default
-configuration.
+The command deploys Elasticsearch on the Kubernetes cluster in the default configuration.
+
+## Integrating with Kanister
+
+In case, if you don't have `Kanister` installed already, you can use following commands to do that.
+Add Kanister Helm repository and install Kanister operator
```

Comment:
```suggestion
In case you don't have `Kanister` installed already, you can use following
commands to do that.
Add Kanister Helm repository and install Kanister operator
```

Reply:
fixed